modules/c++/mt/include/mt/ThreadGroup.h

```diff
@@ -89,13 +85,31 @@ class ThreadGroup
      */
     void joinAll();
 
+    /*!
+     * Checks whether CPU pinning is set (i.e. whether mAffinityInit is NULL)
+     *
+     * \returns true if CPU pinning is set, false otherwise
+     */
+    bool isPinToCPUEnabled() const;
+
+    static bool getDefaultPinToCPU();
+
+    static void setDefaultPinToCPU(const bool newDefault);
+
 private:
     std::auto_ptr<CPUAffinityInitializer> mAffinityInit;
     size_t mLastJoined;
     std::vector<mem::SharedPtr<sys::Thread> > mThreads;
     std::vector<except::Exception> mExceptions;
     sys::Mutex mMutex;
 
+    /*!
+     * The default setting for pinToCPU, used in the ThreadGroup constructor
+     * Can't set non-const static values in the header, so set at the top of
+     * the implementation
+     */
+    static bool defaultPinToCPU;
```

Comment:
For static or constant variables, go with underscore and all caps - `DEFAULT_PIN_TO_CPU`